shared/src/main/scala/com/raquo/domtypes/defs/reflectedAttrs/ReflectedHtmlAttrDefs.scala

```diff
@@ -1105,6 +1105,27 @@ object ReflectedHtmlAttrDefs {
       docUrls = Nil,
     ),
 
+    ReflectedHtmlAttrDef(
+      scalaName = "crossorigin",
```

Comment:
This should be camelCased to follow our naming conventions: "crossOrigin"

Reply:
fixed